elasticsearch/tasks/es-Redhat.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Ensure prerequisites are installed
+  yum:
+    name: "{{ item }}"
+    state: present
+  loop:
+    - java-1.8.0-openjdk
```

Comment:
why we using java 8 only and why this task is not configured for debian based servers